backend/app/models.py

```diff
@@ -1,10 +1,10 @@
-from sqlalchemy import Boolean, Column, Integer, LargeBinary, String, DateTime, ForeignKey, Table
-from sqlalchemy.types import ARRAY
+from sqlalchemy import Boolean, Column, Integer, LargeBinary, String, DateTime, ForeignKey
 from .database import Base
+from sqlalchemy.orm import relationship
 
 
 class User(Base):
-    __tablename__ = "users"
```

Comment:
Looks like some table names are singular, and others are plural. @petersteele111 preference?

Reply:
Just updated it so they are all singular which matches the db model